### ISSUE TITLE
Publish

### ISSUE THREAD

**Issue by flexzuu** [haytko/apollo-link-watched-mutation#2]
Hey I really like the idea of the package. 
What are your plans? Do you want to publish this on npm?

**Comment by ecerroni** [comment]
I agree, being able to manage mutation - query relationships from a central point is an awesome idea.

+1 for the npm package.

**Comment by haytko** [comment]
Wow, I am a poor open-source developer. Sorry about the late reply.

Happy to hear you guys are interested. Honestly I threw this whole thing together pretty quickly for my GrahpQLMeetup talk and haven't gotten around to it since then.

To your question: I definitely want to publish it on npm.

Now that I know people are interested I'll try to devote some time this week to getting it out there. 👍 I'm really interested in hearing how it works for other peoples' use cases.

**Comment by haytko** [comment]
Just published a 0.1.0 version here: https://www.npmjs.com/package/apollo-link-watched-mutation 👍 

Please give it a try and let me know what you think. I've included some debug && readOnly options so you can see exactly what the link is doing & would try to write to the cache, which makes playing around pretty easy.

e.g.

```
new WatchedMutationLink(cache, {
    SaveTodo: {
      TodoList: ({ mutation, query }) => {
        console.log(mutation);
        console.log(query);
        return 'foo';
      }
    }
  }, 1, 1)
```
